### PR TITLE
children

### PR DESCRIPTION
**Pull request by wvffle**
children

### CODE REVIEW

File: src/content/component.js [hunk 1]
```diff
@@ -16,15 +16,15 @@
 
 function Component(element) {
     this.element = element;
-    this.childs = {};
+    this.children = {};
```
Comment (wvffle) [inline src/content/component.js:19]: children

File: src/content/component.js [hunk 2]
```diff
@@ -16,15 +16,15 @@
...
 }
 
 Component.prototype.addChild = function (component) {
-    this.childs.push(component);
+    this.children.push(component);
```
Comment (wvffle) [inline src/content/component.js:23]: children

File: src/content/component.js [hunk 3]
```diff
@@ -16,15 +16,15 @@
 
 function Component(element) {
     this.element = element;
-    this.childs = {};
+    this.children = {};
 }
 
 Component.prototype.addChild = function (component) {
-    this.childs.push(component);
```
Comment (wvffle) [inline src/content/component.js:23]: children

File: src/content/component.js [hunk 4]
```diff
@@ -16,15 +16,15 @@
 
 function Component(element) {
     this.element = element;
-    this.childs = {};
```
Comment (wvffle) [inline src/content/component.js:19]: children

**Review by wvffle — COMMENTED** [review]
children

**Comment by wvffle** [comment]
children

**Comment by MrGregorix** [comment]
childs

**Comment by artur9010** [comment]
children!

**Pull request merged**